package.json

```
@@ -107,5 +107,8 @@
   },
   "workspaces": [
     "packages/*"
-  ]
+  ],
+  "resolutions": {
+    "write-file-atomic": "2.4.1"
+  }
```

We could add something like this to keep track on the error. Maybe creating an issue in gatsby and tagging it with upstream is a good place too 😛 
```suggestion
  },
  "comments": {
    "resolutions": {
      "write-file-atomic": "Jest bug on windows: https://github.com/facebook/jest/issues/4444"
    }
  }
```

---

Documented in https://github.com/gatsbyjs/gatsby/issues/16885